src/app/launcher/create-app/project-summary-createapp-step/project-summary-createapp-step.component.html

```diff
@@ -224,12 +224,15 @@ <h2 class="card-pf-title">
                 </div>
                 <div class="form-group f8launcher-project-summary-data-form-group">
                   <label class="col-xs-4 control-label">Space Path</label>
-                  <div class="col-xs-8 f8launcher-project-summary-data-field f8launcher-application-text-field">
+                  <div class="col-xs-8 f8launcher-project-summary-data-field">
+                    <span>{{launcherComponent?.summary?.dependencyCheck?.spacePath}}</span>
+                  </div>
+                  <!-- <div class="col-xs-8 f8launcher-project-summary-data-field f8launcher-application-text-field">
                     <input class="f8launcher-project-summary-data-field_input f8launcher-application-text-field_input"
                            name="spacePath" placeholder="Space Path" type="text"
                            [(ngModel)]="launcherComponent.summary.dependencyCheck.spacePath"
                            (keyup.enter)="$event.target.blur();">
```

Comment:
Is the path editable for Launcher? It wasn't clear why this feature was added originally. 

It doesn't make sense for OSIO, so I'm fine with removing it. Just want to double check

Reply:
I made it non editable as we are already in a space name and we dont want the user to edit it again in the summary page

Reply:
That's fine.

Reply:
Space path doesn't exist in the standalone Launcher, just OSIO